javascript/javascript3/week3/shoppingCart_main.js

```diff
@@ -0,0 +1,128 @@
+//Shopping cart using Classes
+const body = document.querySelector("body");
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
```

Comment:
Great logic ! However, to make it more flexible also for the future if the conversion rates change you could be using an API call to do the heavy lifting. 

```
convertToCurrency(currency){
 fetch(
        `https://api.exchangerate.host/convert?from=DKK&to=${currency}&amount=${this.price}`
      )
        .then((response) => response.json())
        .then((data) => resolve(data));
}
```